libbeat/api/routes.go

```diff
@@ -47,6 +48,14 @@ func NewWithDefaultRoutes(log *logp.Logger, config *common.Config, ns lookupFunc
 	return New(log, mux, config)
 }
 
+func (s *Server) AttachPprof() {
+	s.log.Info("Attaching pprof endpoints")
+	s.mux.HandleFunc("/debug/pprof/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is serving to 0.0.0.0, correct? Does it want to be limited?

Reply:
The default binding for the libbeat/api server is `localhost:5066`.
I'm not sure what you mean by wanting it to be limited, can you elaborate please?

Reply:
Thanks. You've clarified; I just wanted to make sure the handler is only serving to localhost.